internal/command/lint/command.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/spf13/viper"
+
```

Comment:
Looks like this repo uses 2 import groups: standard library and everything else.
Keep these import groups.